simphony/io/tests/test_h5_lattice.py

```diff
@@ -73,9 +108,10 @@ def cleanup(self):
         shutil.rmtree(self.temp_dir)
 
     def container_factory(self, name, type_, base_vect, size, origin):
-        return H5Lattice.create_new(
-            self.handle.root, 'foo', type_, base_vect,
-            size, origin, record=CustomRecord)
+        self.group = self.handle.create_group(
```

Comment:
If handle.root is used as the group where tables are created, then to set the name properly, container_factory has to set the name of the root group to 'name', because H5Lattice doesn't have the name parameter anymore. Do you think it's better to change the name, than create a subgroup?